public/app/features/panel/metrics_panel_ctrl.ts

```diff
@@ -166,6 +166,23 @@ class MetricsPanelCtrl extends PanelCtrl {
       return this.$q.when([]);
     }
 
+    var targets = JSON.stringify(this.panel.targets);
+    var intervalVar = null;
+    _.each(this.templateSrv.variables, function(variable) {
```

Comment:
The JSON format of targets is differed among data source plugin, so I use JSON.stringify() to check query expression easily.